Cubical/HITs/Truncation/Base.agda

```diff
@@ -1,13 +1,11 @@
 {-# OPTIONS --cubical --safe #-}
 module Cubical.HITs.Truncation.Base where
 
-open import Cubical.Data.NatMinusOne
 open import Cubical.Foundations.Prelude
-open import Cubical.Foundations.HLevels
+open import Cubical.Data.NatMinusOne using (ℕ₋₁; neg1; suc)
+open import Cubical.Data.NatMinusTwo
+open import Cubical.HITs.Nullification
 open import Cubical.HITs.Sn
 
-data  ∥_∥_ {ℓ} (A : Type ℓ) (n : ℕ₋₁) : Type ℓ where
-  ∣_∣ : A  → ∥ A ∥ n
-  hub : (f : S (1+ n) → ∥ A ∥ n) → ∥ A ∥ n
-  spoke : (f : S (1+ n) → ∥ A ∥ n) (x : S (1+ n)) → hub f ≡ f x
-
+∥_∥_ : ∀ {ℓ} → Type ℓ → ℕ₋₂ → Type ℓ
```

Comment:
I think we should keep the old n-truncation code. It might have better/worse computational capabilities, so it would be nice to be able to easily switch definition and do benchmarks. Can you please put the old code in a module?

Reply:
This goes also for the proofs about it in the Properties file

Reply:
I moved the old code to `Truncation/FromNeg1/` and added an explanation of the difference to `Truncation.Base`. Is something like this what you had in mind?

Reply:
Yes, exactly!